docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md

```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
-  <LinkCard topIcon label="Create a Custom Multicolumn Map Expectation" description="Create a Custom Multicolumn Map Expectation that evaluates a set of columns and performs a yes or no query about the row-wise relationship between the the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Create a Custom Regex-Based Column Map Expectation" description="Create a Custom Regex-Based Column Map Expectation that evaluates a single column and performs a yes or no regex-based query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Create a Custom Set-Based Column Map Expectation" description="Create a Custom Set-Based Column Map Expectation that evaluates a single column and determines if each row in the column belongs to the specified set" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Create a Custom Query Expectation" description="Create a Custom Query Expectation to run Expectations against custom query results and make intermediate queries to your database" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="Create a Custom Parameterized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="Use auto-initializing Expectations to automate Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />
-</LinkCardGrid>
\ No newline at end of file
+  <LinkCard topIcon label="Create a Custom Column Aggregate Expectation" description="evaluates a single column and produces an aggregate Metric" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations" icon="/img/custom_expectation_icon.svg" />
+  <LinkCard topIcon label="Create a Custom Column Map Expectation" description="evaluates a single column and performs a yes or no query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Comment:
```suggestion
  <LinkCard topIcon label="Create a Custom Column Map Expectation" description="Evaluates a single column and performs a yes or no query on every column row " href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Reply:
capitalized all first letters in 0dc8a25de